docs/api.md

```diff
@@ -354,6 +355,73 @@ will be closed.
 If no URLs are specified, this method returns all cookies.
 If URLs are specified, only cookies that affect those URLs are returned.
 
+#### browserContext.exposeFunction(name, playwrightFunction)
+- `name` <[string]> Name of the function on the window object.
+- `playwrightFunction` <[function]> Callback function which will be called in Playwright's context.
+- returns: <[Promise]>
+
+The method adds a function called `name` on the `window` object of every page in the context.
+When called, the function executes `playwrightFunction` in node.js and returns a [Promise] which resolves to the return value of `playwrightFunction`.
+
+If the `playwrightFunction` returns a [Promise], it will be awaited.
+
+See [page.exposeFunction(name, playwrightFunction)](#pageexposefunctionname-playwrightfunction) for page-only version.
+
+> **NOTE** Functions installed via `page.exposeFunction` survive navigations.
+
+An example of adding an `md5` function to all pages in the context:
+```js
+const { firefox } = require('playwright');  // Or 'chromium' or 'webkit'.
+const crypto = require('crypto');
+
+(async () => {
+  const browser = await firefox.launch();
+  const context = await browser.newContext();
+  await context.exposeFunction('md5', text =>
+    crypto.createHash('md5').update(text).digest('hex')
+  );
+  const page = await context.newPage();
+  page.on('console', msg => console.log(msg.text()));
+  await page.evaluate(async () => {
```

Comment:
It would be nice if md5 was used in a page context snippet.

Reply:
Done.